### DISCUSSION TITLE
Custom Shortcuts

### DISCUSSION THREAD

**Discussion by steventohme** [mdx-editor/editor#387] [Q&A]
Is there a way to have custom shortcuts, or change the shortcut bindings that are in place?

**Comment by frxderic** [comment]
Hi - are there any news on this topic? I would also like to be able to use custom shortcuts in the editor, e.g. == for highlighting (together with a Custom Directive for example)! Is there a way to achieve this?

**Comment by cklaffer** [comment]
+1
